src/StreamingDataFrames/streamingdataframes/models/serializers/quix.py

```diff
@@ -239,12 +325,31 @@ class QuixTimeseriesSerializer(JSONSerializer):
 
     """
 
-    # Quix Timeseries data must have the following headers set
     extra_headers = {
         QModelKey.HEADER_NAME: QModelKey.TIMESERIESDATA,
         QCodecId.HEADER_NAME: QCodecId.JSON_TYPED,
     }
 
+    def __init__(
+        self,
+        as_legacy: Optional[bool] = None,
```

Comment:
Similar, here looks like it can be `as_legacy: bool = True` vs `None` and extra check in __init__